protocol/scripts/deploys/mainnet/v2/ramos/01-temple-dai-balancer-pool.ts

```diff
@@ -1,11 +1,11 @@
 import '@nomiclabs/hardhat-ethers';
 import { ethers, network } from 'hardhat';
-import { FakeERC20__factory, IERC20 } from '../../../../typechain';
+import { FakeERC20__factory, IERC20 } from '../../../../../typechain';
 import {
     ZERO_ADDRESS,
     ensureExpectedEnvvars,
     mine,
-} from '../../helpers';
+} from '../../../helpers';
 import { JsonRpcProvider } from "@ethersproject/providers"
 import { BalancerSDK, PoolType, WeightedCreatePoolParameters } from '@balancer-labs/sdk';
 import { BigNumber, Signer } from 'ethers';
```

Comment:
Creating new pools isn't required for mainnet - we'll just use the existing RAMOS DAI pool.

You should be able to copy the balancer/aura dependencies from here instead:
https://github.com/TempleDAO/temple/blob/stage/protocol/scripts/deploys/helpers.ts#L355